docs/reference/migration/migrate_5_6.asciidoc

```diff
@@ -33,3 +33,8 @@ previous behavior, Elasticsearch would skip the bootstrap checks if bound to a
 link-local address. Now that Elasticsearch no longer binds to such addresses
 when binding to localhost, the bootstrap checks now apply to link-local
 addresses.
+
+[float]
+=== Referencing Parent Documents
+
+Previously in Elasticsearch parent documents could be aggregated on or referenced in a script using `_parent`. Now in order to reference a parent document in an aggregation or script you must do `_parent#my_parent` as stated in the https://www.elastic.co/guide/en/elasticsearch/reference/5.6/mapping-parent-field.html#CO211-2[mapping a parent field documentation].
```

Comment:
Thanks for the PR. This should not be a hard link but rather a doc link to the `mapping-parent-field` section in the docs. Can you help with this @jimczi?

Also, would you please wrap the lines at 80-columns?